suite/tests/api/ir_arm.c

```diff
@@ -81,7 +81,10 @@ test_instr_encoding(void *dc, uint opcode, instr_t *instr)
     pc = instr_encode(dc, instr, buf);
```

Comment:
Assert pc != NULL? It was checked in the old version of `test_pcrel`.

Reply:
Done.